README.md

```diff
@@ -9,3 +9,9 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+If the graphs have the same number of nodes and each node is connected to every other node then you can make a map that assigns each vertex in graph 1 to the graph2 vertex so they must be one to one connections.
```

Comment:
No, I can e.g. make a mapping from each node in the first graph to a single node in the second graph.